editor/docks/scene_tree_dock.cpp

```diff
@@ -1729,11 +1729,6 @@ void SceneTreeDock::_notification(int p_what) {
 
 			filter->set_right_icon(get_editor_theme_icon(SNAME("Search")));
 
-			PopupMenu *filter_menu = filter->get_menu();
-			filter_menu->set_item_icon(filter_menu->get_item_idx_from_text(TTR("Filters")), get_editor_theme_icon(SNAME("Search")));
-			filter_menu->set_item_icon(filter_menu->get_item_index(FILTER_BY_TYPE), get_editor_theme_icon(SNAME("Node")));
-			filter_menu->set_item_icon(filter_menu->get_item_index(FILTER_BY_GROUP), get_editor_theme_icon(SNAME("Groups")));
```

Comment:
> Icons are removed. Together with the checkbox above, they create confusing empty spaces.

They don't have to since https://github.com/godotengine/godot/pull/112545 . As the original author, I do like the icons...